landing.css

```diff
@@ -0,0 +1,126 @@
+@import "./base.css";
+@import "./utility.css";
+
+/* Navigation bar */
+
+.brand-name{
+    font-size: 2rem;
+    padding-bottom: 0.5rem;
+}
+.brand-logo{
+    margin-left: 1.6rem;
+    margin-top:1.2rem;
+}
+.nav-bar-hero{
+    display:flex
+}
+.nav-header{
+    background-color: var(--secondary-color) ;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: var(--text-white-color);
+}
+
+.nav-bar-tabs{
+    display: flex;
+    justify-content: space-around;
+    width:15rem;
+}
+.nav-text{
+    margin-top: 2rem;
+    font-weight: 600;
+}
+.nav-tab-text{
+    color:var(--primary-color);
+}
+.nav-tab-text:hover{
+cursor: pointer;
+color: var(--text-white-color);
+ ;
+}
+
+/* Hero section */
+
+.hero-img{
+    height: 28rem;
+    margin:2rem;
+}
+.hero-img-container{
+    display:flex;
+}
+
+.hero-text-container{
+    margin-top: 6rem;
+}
+
+.fs-2{
+    font-size: 1.2rem;
+}
+.brand-text{
+font-weight: bold;
+font-size: 1.5rem;
+}
+
+.btn-primary{
+    margin :1rem 0.5rem;
+    padding:0.5rem;
+    background-color: var(--primary-color);
+    border-radius: 0.25rem;
+    border-color:var(--primary-color) ;
+    font-weight: bold;
+}
+.btn-primary:hover{
+    cursor: pointer;
+}
+
+
+/* colors */
+
+.white-background{
+    background-color: var(--text-white-color);
+}
+.orange-theme-color{
+    color: var(--primary-color);
+}
+.orange-theme-dark-color{
+    color:var(--primary-color)
+}
+.white-text-color{
+    color:var(--text-white-color) ;
+}
+
+/* Footer */
+
+.footer{
+    background-color: var(--secondary-color);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-text{
+    padding: 0.5rem;
```

Comment:
Recommendation to use `px` for any value less than 12 or 16px, as suggested in CSS sessions

Reply:
> Recommendation to use `px` for any value less than 12 or 16px, as suggested in CSS sessions

Sure, will do that